lib/spack/spack/install_test.py

```diff
@@ -3,32 +3,73 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 import base64
+import contextlib
+import enum
 import hashlib
+import inspect
+import io
 import os
 import re
 import shutil
+import sys
+from collections import Counter
+from typing import Callable, List, Optional, Tuple, Type, Union
 
 import llnl.util.filesystem as fs
 import llnl.util.tty as tty
+from llnl.util.lang import nullcontext
+from llnl.util.tty.color import colorize
 
 import spack.error
 import spack.paths
 import spack.util.spack_json as sjson
 from spack.spec import Spec
 from spack.util.prefix import Prefix
+from spack.util.string import plural
 
+#: Stand-alone test failure info type
+TEST_FAILURE_TYPE = Tuple[BaseException, str]
```

Comment:
Can we adhere to `pylint` style, i.e. use camel case for types:
```python
TestFailureType = Tuple[BaseException, str]
```
and capital letters for global variables:
```python
TEST_SUITE_FILENAME = "test.suite.lock"
```
? If we use single standard it would help reading code (since the name would convey information on the associated variable).

Reply:
Sure.  I initially re-used the case I'd seen someone else do elsewhere (e.g., `package_base.py`'s use for `FLAG_HANDLER_[RETURN_]TYPE`).